lib/modules/wdpa/global_stats_importer.rb

```diff
@@ -4,14 +4,16 @@ module Wdpa::GlobalStatsImporter
   GLOBAL_STATS_CSV = Rails.root.join('lib/data/seeds/global_stats.csv').freeze
 
   def self.import
+    GlobalStatistic.destroy_all
+
     attrs = {singleton_guard: 0}
     CSV.foreach(GLOBAL_STATS_CSV, headers: true) do |row|
       field = row['type']
       value = parse_value(row['value'])
       attrs.merge!("#{field}": value)
     end
-
-    GlobalStatistic.create(attrs)
+    
+    GlobalStatistic.first_or_create(attrs)
```

Comment:
If you go for the `destroy_all` option above, you could just leave `create` as `first` will always be nil anyway

Reply:
Left that in by mistake - how about this new method? Should now update in place if it differs